### PR TITLE
Merge 17.0 code freeze into develop

### PR DESCRIPTION
**Pull request by jkmassel**
Merges the 17.0 code freeze into `develop`.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -341,4 +341,4 @@ DEPENDENCIES
   xcpretty-travis-formatter
 
 BUNDLED WITH
-   2.2.10
+   2.2.14
```
Comment (mokagio) [inline Gemfile.lock:344]: They released 2.2.15, if you really want to be on the latest 😛

File: Podfile [hunk 2]
```diff
@@ -171,8 +171,8 @@ target 'WordPress' do
     ##
     pod 'Charts', '~> 3.2.2'
     pod 'Gifu', '3.2.0'
-    pod 'AppCenter', '2.5.1', :configurations => ['Release-Internal', 'Release-Alpha']
-    pod 'AppCenter/Distribute', '2.5.1', :configurations => ['Release-Internal', 'Release-Alpha']
+    pod 'AppCenter', '4.1.1', :configurations => ['Release-Internal', 'Release-Alpha']
+    pod 'AppCenter/Distribute', '4.1.1', :configurations => ['Release-Internal', 'Release-Alpha']
```
Comment (mokagio) [inline Podfile:175]: This is quite the version bump.

Was it due to the Fastlane plugin update? If so, weird that they did it on a patch version bump .

File: WordPress/Classes/System/WordPressAppDelegate.swift [hunk 3]
```diff
@@ -401,8 +401,7 @@ extension WordPressAppDelegate {
 
     @objc func configureAppCenterSDK() {
         #if APPCENTER_ENABLED
-        MSAppCenter.start(ApiCredentials.appCenterAppId(), withServices: [MSDistribute.self])
-        MSDistribute.setEnabled(true)
+        AppCenter.start(withAppSecret: ApiCredentials.appCenterAppId(), services: [Distribute.self])
```
Comment (mokagio) [inline WordPress/Classes/System/WordPressAppDelegate.swift:404]: This looks good to me.

I guess the only way to actually test it, though, is to ship another build to App Center and see if the this one shows the update prompt and if the process succeeds?

**Review by mokagio — CHANGES_REQUESTED** [review]
GTG once CI let's us 👍

**Review by mokagio — APPROVED** [review]

**Comment by mokagio** [comment]
(_I didn't mean to request changes..._)

File: Podfile (hunk 2 above)
Comment (jkmassel) [reply Podfile:175]: I did this by hand to get it up to date 😃

File: WordPress/Classes/System/WordPressAppDelegate.swift (hunk 3 above)
Comment (jkmassel) [reply WordPress/Classes/System/WordPressAppDelegate.swift:404]: Yeah, exactly – I'm sure we'll have one this release 😅

**Pull request merged**